tests/e2e/s4l/sim4life-dy-autoscale.js

```diff
@@ -0,0 +1,78 @@
+// node sim4life.js [url] [user] [password] [timeout] [--demo]
+
+const utils = require('../utils/utils');
+const tutorialBase = require('../tutorials/tutorialBase');
+
+const args = process.argv.slice(2);
+const {
+  url,
+  user,
+  pass,
+  newUser,
+  nUsers,
+  userPrefix,
+  userSuffix,
+  startTimeout,
+  enableDemoMode
+} = utils.parseCommandLineArguments(args)
+
+const serviceName = "sim4life-dy";
+
+async function runTutorial(user, pass, newUser, parallelUserIdx) {
+  const tutorial = new tutorialBase.TutorialBase(url, serviceName, user, pass, newUser, enableDemoMode, parallelUserIdx);
+  let studyId;
+  try {
+    await tutorial.start();
+
+    // start sim4life-dy service
+    const studyData = await tutorial.openService(1000);
+    studyId = studyData["data"]["uuid"];
+
+    const workbenchData = utils.extractWorkbenchData(studyData["data"]);
+    console.log(workbenchData);
+    const s4lNodeId = workbenchData["nodeIds"][0];
+    await tutorial.waitForServices(
+      workbenchData["studyId"],
+      [s4lNodeId],
+      startTimeout,
+      false
+    );
+
+    // NOTE: we do not test anything but service being ready here
```

Comment:
```suggestion
    await tutorials.testS4L(s4lNodeId)
```

It will just check that the modeling is properly loaded

Reply:
yes but there is no license server so it does not work right now